src/host/config/KBoxConfigParser.cpp

```diff
@@ -46,7 +46,15 @@ void KBoxConfigParser::defaultConfig(KBoxConfig &config) {
   config.serial2Config.inputMode = SerialModeNMEA;
   config.serial2Config.outputMode = SerialModeDisabled;
 
-  config.imuConfig.enabled = true;
+  config.imuConfig.enabled = true;            // enable internal IMU sensor
+  config.imuConfig.enabledHdg = true;         // true if values taken from internal sensor
+  config.imuConfig.enabledHeelPitch = true;   // true if values taken from internal sensor
+  config.imuConfig.calHdg = 2;                // hdg valid, if calibration value greater equal
+  config.imuConfig.calHeelPitch = 2;          // heel (roll) & pitch valid, if calibration value greater equal
+  config.imuConfig.writeN2kHdg = false;       // write heading to N2k bus
```

Comment:
I was thinking we would do this in the NMEA2000 configuration. For example, we would have: 
```
config.nmea2000.attitudePGN = true
```
to turn on generation of this PGN.

This way, no matter how we received it, you can decide if you want to send it to NMEA2000.

I do not know which one is best. Just sharing for discussion. We can start like this and see where it takes us.

Reply:
Consider an external DIY-Sensor (my friend is working on this with a NXP-IMU-sensor) which is plugged to serial input. 

I asked myself: "How would it be possible to write HeadingPGN from external sensor only (not from internal IMU), but write attitude from internal sensor?"

But you are right, with settings:
```
config.imuConfig.enableHeelPitch = true;
config.imuConfig.enableHdg = false;
```
it would be enough to to it in the NMEA2000 config.

So I will take out  ```config.imuConfig.writeN2kHdg```

(I do not think that this settings will ever be on a web based config page (because they are too special) but I believe that there should be in the config file.)